src/main/java/com/google/cloud/spanner/hibernate/SpannerDialect.java

```diff
@@ -90,8 +90,8 @@ public SpannerDialect() {
     registerColumnType(Types.NCLOB, "STRING(MAX)");
     registerColumnType(Types.BLOB, "BYTES(MAX)");
 
-    registerColumnType(Types.DECIMAL, "FLOAT64");
-    registerColumnType(Types.NUMERIC, "FLOAT64");
+    registerColumnType(Types.DECIMAL, "STRING(MAX)");
```

Comment:
Do you intend to include these changes with the sequence stuff?

Reply:
reverted. I guess we should wait until we have a driver that supports these two types before we make changes